onnx_chainer/export.py

```diff
@@ -144,41 +137,41 @@ def backward_postprocess(self, function, in_data, out_grad):
                 output_name = self.context.get_name(o())
             output_names.append(output_name)
 
-        opset_versions = mapping.operators[func_name]
-        if isinstance(opset_versions, int):
-            opset_version = opset_versions
-        elif self.specified_opset_version is None:
-            # If no opset version is specified,
-            # use the latest version for the operator
-            opset_version = opset_versions[-1]
-        else:
-            # If a version is specified, use the last version <= specified one
-            for opset_version in sorted(opset_versions, reverse=True):
-                if opset_version <= self.specified_opset_version:
-                    break
-
-        if opset_version > self.specified_opset_version:
-            raise RuntimeError('ONNX-chainer cannot convert `{}` of Chainer '
-                               'with ONNX opset_version {}'.format(
-                                   func_name, self.specified_opset_version))
-
-        nodes = create_node(
-            func_name, opset_version, function, input_names,
-            output_names, self.context, self.additional_parameters)
+        nodes = self.create_node(
+            func_name, function, input_names, output_names,
+            self.additional_parameters)
         self.graph.extend(nodes)
 
 
 def export(model, args, filename=None, export_params=True,
            graph_name='Graph', save_text=False, opset_version=None,
-           train=False, return_flat_inout=False):
+           train=False, return_flat_inout=False, external_converters=None,
+           external_opset_imports=None):
     """Export function for chainer.Chain in ONNX format.
 
     This function performs a forward computation of the given
     :class:`~chainer.Chain`, ``model``, by passing the given arguments ``args``
     directly. It means, the output :class:`~chainer.Variable` object ``y`` to
     make the computational graph will be created by:
 
-    y = model(*args)
+    ``y = model(*args)``
+
+    ``external_converters`` and ``external_opset_import`` are for external
+    custom operator. When some ~chainer.FunctionNode are expected to convert to
+    own customized operator, set converter function with ~chainer.FunctionNode
+    name.
+
+    >>> def custom_converter(param):
+    >>>     return onnx.make_node('CustomizedOp', domain='chainer'),
```

Comment:
I think this sample just makes an orphan node which is not actually very useful. Could you use `input_names` and `output_names` in this example?